### PR TITLE
docs: Add how-to for integrating with vw-estimators

### PR DESCRIPTION
**Pull request by jackgerrits**
Requires #48

### CODE REVIEW

**Review by ataymano — APPROVED** [review]

File: docs/source/how_to_guides/cb_estimators.ipynb [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    "estimator = ips.Estimator()\n",
+    "interval = cressieread.Interval(empirical_r_bounds=True)\n",
+    "\n",
+    "estimates = []\n",
+    "lower = []\n",
+    "upper = []\n",
+    "\n",
+    "with open(\"example_cb.txt\") as dataset_file:\n",
+    "    with vw.TextFormatReader(workspace, dataset_file) as reader:\n",
+    "        for example in reader:\n",
```
Comment (ataymano) [inline docs/source/how_to_guides/cb_estimators.ipynb]: "example" here vs "examples" in the next line (extract_label signature) is a bit confusing

Comment (jackgerrits) [reply docs/source/how_to_guides/cb_estimators.ipynb]: True. Perhaps I should rename to event?

Comment (ataymano) [reply docs/source/how_to_guides/cb_estimators.ipynb]: yeah I think it is better

File: docs/source/how_to_guides/cb_estimators.ipynb [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    "            continue\n",
+    "\n",
+    "        # 2. Predict and learn on the event\n",
+    "        pmf = workspace.predict_then_learn_one(event)\n",
+    "\n",
+    "        # 3. Extract the logged cost and the probability of choosing it according to the logged policy\n",
+    "        logged_action_0_based, logged_cost, logged_prob = logged_label\n",
+    "\n",
+    "        # 4. Get the probability of choosing the logged action according to the target policy\n",
+    "        prediction_prob = next(x for i, x in pmf if i == logged_action_0_based)\n",
```
Comment (ataymano) [inline docs/source/how_to_guides/cb_estimators.ipynb:120]: [nit] dict(pmf)[logged_action_0_based] (not sure which one is more readable)

**Review by ataymano — APPROVED** [review]

File: docs/source/how_to_guides/cb_estimators.ipynb (hunk 2 above)
Comment (jackgerrits) [reply docs/source/how_to_guides/cb_estimators.ipynb:120]: Oh interesting, I didn't know you could init a dictionary like that

**Pull request merged**